demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -275,17 +275,17 @@ def test_is_root_connected_to_all_tasks(self, source, answer):
         finally:
             os.remove(PLAYBOOK_TARGET)
 
-    IS_ALL_PARAMS_NOT_HIDDEN_INPUTS = [
+    IS_VALID_HIDDEN_PARAMS = [
         (VALID_NO_HIDDEN_PARAMS, True),
```

Comment:
i dont understand how this is valid, it has hidden params, which are not `longRunning`

Reply:
it doesn't, those are `hidden: false` params